homeassistant/helpers/state.py

```diff
@@ -120,6 +120,32 @@ def get_changed_since(states, utc_point_in_time):
             if state.last_updated >= utc_point_in_time]
 
 
+def async_get_service(hass, service_domain, state):
```

Comment:
We should never import from components and use things like `hass.components.light.is_on` instead